rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -185,20 +184,20 @@ components:
           summary: A development version
     Location:
       description: >
-        The URI that can be used to download the file for the Product.
+        The URI that can be used to download the file for the dataset.
       required: true
       schema:
         type: string
         format: uri
         minLength: 1
         maxLength: 500
   schemas:
-    File:
+    Attachment:
       type: object
       properties:
         file:
           description: >
-            The file attachment. The current maximum size is the character equivalent of 10GB.
+            The attachment. The current maximum size is the character equivalent of 10GB.
```

Comment:
dataset attachment? metadata attachment? 
I feel like we need at least one more word to describe what kind of attachment this is...

Reply:
It's both. This schema is used for the both file and metadata attachments. If it's confusing to reuse this schema, I can inline it in the 2 places that it's used.